setup/native-script.ps1

```diff
@@ -1,5 +1,9 @@
 
 # A Boxstarter script to set up Windows machine for NativeScript development
+# To run it against RELEASE branch (recommended) use
```

Comment:
maybe it is worth using the 'production' branch (that we've discussed for our docs) here as well